src/include/symbol_table/symbole_table.h

```diff
@@ -0,0 +1,101 @@
+#ifndef JUCC_SYMBOL_TABLE_SYMBOL_TABLE_H
+#define JUCC_SYMBOL_TABLE_SYMBOL_TABLE_H
+
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+namespace jucc {
```

Comment:
Use namespace symboltable

Reply:
Why ? 
Everything was supposed to be within namespace jucc !

Reply:
jucc::symbol_table, double namespace

Reply:
yeah doing it!